sdk/storage/tests.yml

```diff
@@ -12,6 +12,8 @@ extends:
       Preview:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
         ServiceConnection: azure-sdk-tests
+        SubscriptionConfigurationFiles:
+          - sdk/storage/sub-config-azure-preview.json
```

Comment:
This path does not match the path of the file created: sdk/storage/storage-preview-public-msft.json